tests/backend/mongodb/test_queries.py

```diff
@@ -0,0 +1,367 @@
+import pytest
+
+pytestmark = [pytest.mark.bdb]
```

Comment:
Just a small detail: the list is not needed for only one marker, i.e.:

```python
pytestmark = pytest.mark.bdb
```

would also work.